# HG changeset patch
# User Rik <rik@octave.org>
# Date 1360819710 28800
#      Wed Feb 13 21:28:30 2013 -0800
# Node ID f4f0aea29b2167a4c3644b2ecebe7734dc0c3ecf
# Parent  5306e943ce74f2a73fcc26776a8cbb03e99c2d6d
build: remove *-tst files with distclean rule.

* libinterp/Makefile.am: Add $(TST_FILES) to distclean rule

* liboctave/Makefile.am: Add $(TST_FILES) to distclean rule

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -351,12 +351,14 @@ CLEANFILES = \
   interpfcn/graphics-props.cc \
   parse-tree/oct-parse.output
 
 DISTCLEANFILES = \
   .DOCSTRINGS \
   DOCSTRINGS \
   $(BUILT_NODISTFILES) \
   $(OCT_FILES) \
-  $(OCT_STAMP_FILES)
+  $(OCT_STAMP_FILES) \
+  $(TST_FILES)
 
 MAINTAINERCLEANFILES = \
   $(BUILT_DISTFILES)
+
diff --git a/liboctave/Makefile.am b/liboctave/Makefile.am
--- a/liboctave/Makefile.am
+++ b/liboctave/Makefile.am
@@ -130,10 +130,12 @@ LIBOCTAVE_TST_SRC = \
 TST_FILES_SRC := $(shell $(top_srcdir)/build-aux/find-files-with-tests.sh "$(srcdir)" $(LIBOCTAVE_TST_SRC))
 
 TST_FILES := $(addsuffix -tst,$(TST_FILES_SRC))
 
 liboctavetestsdir := $(octtestsdir)/liboctave
 
 nobase_liboctavetests_DATA = $(TST_FILES)
 
-DISTCLEANFILES += $(BUILT_INCS)
+DISTCLEANFILES += \
+  $(BUILT_INCS) \
+  $(TST_FILES)
 
